linode/nodepool/framework_resource_test.go

```diff
@@ -0,0 +1,367 @@
+////go:build integration
+
+package nodepool_test
+
+import (
+	"context"
+	"fmt"
+	acceptanceTmpl "github.com/linode/terraform-provider-linode/v2/linode/acceptance/tmpl"
+	"log"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/linode/terraform-provider-linode/v2/linode/acceptance"
+	"github.com/linode/terraform-provider-linode/v2/linode/helper"
+	"github.com/linode/terraform-provider-linode/v2/linode/nodepool/tmpl"
+
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/linode/linodego"
+)
+
+var (
+	clusterID  string
+	k8sVersion string
+	testRegion string
+)
+
+func init() {
+	resource.AddTestSweepers("linode_nodepool", &resource.Sweeper{
+		Name: "linode_nodepool",
+		F:    sweep,
+	})
+
+	clusterID = os.Getenv("LINODE_TEST_CLUSTER_ID")
+
+	if clusterID == "" {
+		client, err := acceptance.GetTestClient()
+		if err != nil {
+			log.Fatalf("failed to get client: %s", err)
+		}
+
+		versions, err := client.ListLKEVersions(context.Background(), nil)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		k8sVersions := make([]string, len(versions))
+		for i, v := range versions {
+			k8sVersions[i] = v.ID
+		}
+
+		sort.Strings(k8sVersions)
+
+		if len(k8sVersions) < 1 {
+			log.Fatal("no k8s versions found")
+		}
+
+		k8sVersion = k8sVersions[len(k8sVersions)-1]
+
+		region, err := acceptance.GetRandomRegionWithCaps([]string{"kubernetes"})
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		testRegion = region
+	}
+}
+
+func GetTestClient() (*linodego.Client, error) {
+	token := os.Getenv("LINODE_TOKEN")
+	if token == "" {
+		return nil, fmt.Errorf("LINODE_TOKEN must be set for acceptance tests")
+	}
+
+	apiVersion := os.Getenv("LINODE_API_VERSION")
+	if apiVersion == "" {
+		apiVersion = "v4beta"
+	}
+
+	config := &helper.Config{
+		AccessToken: token,
+		APIVersion:  apiVersion,
+		APIURL:      os.Getenv("LINODE_URL"),
+	}
+
+	client, err := config.Client(context.Background())
+	if err != nil {
+		return nil, err
+	}
+
+	return client, nil
+}
```

Comment:
You should be able to directly import `acceptance.GetTestClient(...)` from here 👍 
https://github.com/linode/terraform-provider-linode/blob/dev/linode/acceptance/util.go#L657

Reply:
done